### PR TITLE
Feature: add empty rule condition check, add introduction video and modify UI details

### PR DESCRIPTION
**Pull request by panchongyang**

### CODE REVIEW

File: feature-probe-ui/src/constants/dictionary_keys.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const LAST_SEEN = 'user_last_seen';
 export const DEMO_TIP_SHOW = 'is_demo_tips_show';
-export const USER_GUIDE_LAYOUT = 'user_guide_layout';
+export const USER_GUIDE_LAYOUT = 'user_guide_layout_version_1';
```
Comment (frankzhaopeng) [inline feature-probe-ui/src/constants/dictionary_keys.ts:3]: Why adding a new key here?  can not use `user_guide_layout` directly?

File: feature-probe-ui/src/locales/en-US.json [hunk 2]
```diff
@@ -477,6 +480,8 @@
   "guide.global.step1.enter": "Switch the entry for viewing the toggles of different projects",
   "guide.global.step2.title": "Switch environment",
   "guide.global.step2.env": "Click to switch the environment and view the toggles information in different environments",
+  "guide.global.step3.title": "Segment",
+  "guide.global.step3.env": "Commonly used members can be set up as segment, which can be repeatedly selected in the 'rules' of the switch configuration",
```
Comment (frankzhaopeng) [inline feature-probe-ui/src/locales/en-US.json:484]: repeated

File: feature-probe-ui/src/locales/zh-CN.json [hunk 3]
```diff
@@ -477,6 +479,8 @@
   "guide.global.step1.enter": "切换查看不同项目「开关列表」的入口",
   "guide.global.step2.title": "切换环境",
   "guide.global.step2.env": "点击可切换环境，查看不同环境下的开关信息",
+  "guide.global.step3.title": "人群组",
+  "guide.global.step3.env": "常用人群可以设立为单独的人群组，方便在开关配置的「规则」中反复选用",
```
Comment (frankzhaopeng) [inline feature-probe-ui/src/locales/zh-CN.json:483]: same with last comment

File: feature-probe-ui/src/constants/dictionary_keys.ts (hunk 1 above)
Comment (panchongyang) [reply feature-probe-ui/src/constants/dictionary_keys.ts:3]: Because we should start in first，and this is the easiest way

**Pull request merged**